phpBB/includes/ucp/ucp_resend.php

```diff
@@ -145,19 +145,16 @@ function main($id, $mode)
 					foreach ($messenger_collection_iterator as $messenger_method)
```

Comment:
Ang again, should it be only send by email? If not, probably use notification manager.
May be that is for another ticket/PR though.